bundles/AdminBundle/Resources/public/js/pimcore/object/classes/data/gender.js

```diff
@@ -66,6 +67,9 @@ pimcore.object.classes.data.gender = Class.create(pimcore.object.classes.data.da
         var nameField = this.layout.getComponent("standardSettings").getComponent("name");
         nameField.disable();
 
+        let mandatoryField = this.layout.getComponent("standardSettings").getComponent("mandatory");
+        mandatoryField.disable();
```

Comment:
what about the fields which are already marked as mandatory? I can't make then non-mandatory anymore
<img width="112" alt="image" src="https://github.com/pimcore/pimcore/assets/5137917/954e3da2-c130-4d7b-b612-b42efc8b8203">

Reply:
I have added the option to uncheck the madatory checkbox if it's already marked as a mandatory field in [9cc9709](https://github.com/pimcore/pimcore/pull/15385/commits/9cc9709b04774a09c8af7ef3ba87ab5ba8132b1f)